src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.csproj

```diff
@@ -9,6 +9,30 @@
     <RootNamespace>Dynamo.DocumentationBrowser</RootNamespace>
     <AssemblyName>DocumentationBrowserViewExtension</AssemblyName>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
why do we need all these conditions ?
Can't we just add a single PropertyGroup for all config/platform configs (i.e no condition) ?

ex.
```
<PropertyGroup>
<NoWarn>MSB3539;CS1591;NUnit2005;NUnit2007;CS0618;CS0612;CS0672</NoWarn>
  </PropertyGroup>
```